.github/SECURITY.md

```diff
@@ -0,0 +1,3 @@
+# Security Policy
```

Comment:
Did you intend to put this in the root directory instead?

Reply:
it can go in either and still work (see https://github.com/istio/istio/security/policy). So whatever you prefer

Reply:
Aha. Let's put it in the root directory so it's more visible.  Thank you.